tests/components/evohome/test_init.py

```diff
@@ -23,8 +23,9 @@ async def test_entities(
     """Test entities and state after setup of a Honeywell TCC-compatible system."""
 
     # some extended state attrs are relative the current time
-    freezer.move_to("2024-07-10 12:00:00+00:00")
+    freezer.move_to("2024-07-10T12:00:00Z")
```

Comment:
Why are you changing this time?

Reply:
I used a different format, only to be consistent with the dtm string used across the evohome landscape...

Is there a preferred format to favour?

Reply:
Oh, sorry - I think I misunderstood your question.

The entities have extended state attrs that include scheduled setpoints (i.e. a dtm : target temp pair). Unless the day of week / time of day if fixed here, then they will not likely match those in the snapshots.